server/features/ingest_provider.feature

```diff
@@ -121,6 +121,29 @@ Feature: Ingest Provider
 
     @auth
     @notification
+    Scenario: Update critical errors for ingest_provider
+        Given "ingest_providers"
+	    """
+        [{
+        "type": "reuters",
+        "name": "reuters 4",
+        "source": "reuters",
+        "is_closed": false,
+        "config": {"username": "foo", "password": "bar"}
+        }]
+	    """
+        When we patch "/ingest_providers/#ingest_providers._id#"
+        """
+        {"critical_errors":{"6000":true, "6001":true}}
+        """
+        Then we get updated response
+        """
+        {"critical_errors":{"6000":true, "6001":true}}
+        """
+
+
```

Comment:
I created a unit test for closing the channel when a critical error is thrown. It is on update_ingest_tests.py. I think it covers the closing functionality. Let me know if you see anything missing.